Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js

```diff
@@ -633,20 +635,8 @@ WI.NavigationSidebarPanel = class NavigationSidebarPanel extends WI.SidebarPanel
 
     _checkForStaleResourcesIfNeeded()
     {
-        if (!this._checkForStaleResourcesTimeoutIdentifier || !this._shouldAutoPruneStaleTopLevelResourceTreeElements)
-            return;
-        this.pruneStaleResourceTreeElements();
-    }
-
-    _checkForStaleResources(event)
-    {
-        console.assert(this._shouldAutoPruneStaleTopLevelResourceTreeElements);
```

Comment:
NIT: i think this can stay as-is since we should only reach here from event listeners added inside `if (this._shouldAutoPruneStaleTopLevelResourceTreeElements)`